Changelog.md

```diff
@@ -1,3 +1,10 @@
+## 0.2.7 - 2024-05-06
+
+### Added
+
+-   option to program and evaluate the nRF Cloud Multi-Service Evaluation
+    firmware on nRF9161 DK and nRF9151 DK.
```

Comment:
```suggestion
-   Option to program and evaluate the nRF Cloud Multi-Service Evaluation
    firmware on the nRF9161 DK and the nRF9151 DK.
```

Reply:
Should I not also replace `nRF Cloud Multi-Service` with `nRF Cloud multi-service` here?

Reply:
Yes:
```suggestion
-   Option to program and evaluate the nRF Cloud multi-service 
    firmware on the nRF9161 DK and the nRF9151 DK.
```